datastore/transaction.go

```diff
@@ -110,7 +110,10 @@ func (c *Client) NewTransaction(ctx context.Context, opts ...TransactionOption)
 	return c.newTransaction(ctx, newTransactionSettings(opts))
 }
 
-func (c *Client) newTransaction(ctx context.Context, s *transactionSettings) (*Transaction, error) {
+func (c *Client) newTransaction(ctx context.Context, s *transactionSettings) (_ *Transaction, err error) {
+	ctx = trace.StartSpan(ctx, "cloud.google.com/go/datastore.ReadWriteTransaction")
+	defer func() { trace.EndSpan(ctx, err) }()
+
```

Comment:
`readWrite` is a transaciton type. On 118 line there is a condition `readOnly`. If transaction is not `readOnly` than it's `readWrite`, so it's the right place, the span lines just should be moved under the if statement

Reply:
Done